lib/srv/db/cloud/watchers/rds.go

```diff
@@ -88,7 +88,12 @@ func (f *rdsDBInstancesFetcher) Get(ctx context.Context) (types.Databases, error
 
 // getRDSDatabases returns a list of database resources representing RDS instances.
 func (f *rdsDBInstancesFetcher) getRDSDatabases(ctx context.Context) (types.Databases, error) {
-	instances, err := getAllDBInstances(ctx, f.cfg.RDS, common.MaxPages)
+	instances, err := getAllDBInstances(ctx, f.cfg.RDS, common.MaxPages, false)
+	if common.IsUnrecognizedAWSEngineNameError(err) {
```

Comment:
do you think if we can remember this decision so we don't have to fall back every time?

Reply:
I wanted to avoid the extra complexity of caching, especially since the problem with AWS was intermittent and eventually resolved itself. You're suggesting an optimization, but I don't think it's likely to ever be a performance problem, and we should wait and see if it becomes one before adding a lot of extra logic around caching

Reply:
my bad, i thought this is a permanent thing where a region is retiring old aurora types. seems that's not the case. Then agreed to keep it simple 👍